invenio_rdm_records/services/communities/service.py

```diff
@@ -325,3 +330,27 @@ def search_suggested_communities(
             extra_filter=communities_filter,
             **kwargs,
         )
+
+    @unit_of_work()
+    def set_default(self, identity, id_, data, uow):
+        """Set default community."""
+        valid_data, _ = self.communities_schema.load(
+            data,
+            context={
+                "identity": identity,
+            },
+            raise_errors=True,
+        )
+
+        record = self.record_cls.pid.resolve(id_)
+        self.require_permission(identity, "manage", record=record)
+        record.parent.communities.default = valid_data["default"]
```

Comment:
minor: should we check if a community with the default id exists? Maybe even check if all passed ids exist?

Reply:
Sorry, just saw the other PR where the check happens :)